packages/ai-gemini/tests/parallel-function-response.test.ts

```
@@ -0,0 +1,168 @@
+import { describe, expect, it } from 'vitest'
+import type { ModelMessage } from '@tanstack/ai'
+import type { Content } from '@google/genai'
+import { GeminiTextAdapter } from '../src/adapters/text'
```

ESLint

[error] `../src/adapters/text` import should occur before type import of `@tanstack/ai`

(import/order)